common/src/model_registration/register.py

```diff
@@ -13,10 +13,10 @@
 from azure.ai.ml.entities import Model
 from azureml.core import Run
 from azure.identity import ManagedIdentityCredential
-from pathlib import Path
-
+import yaml
 
 SUPPORTED_MODEL_ASSET_TYPES = [AssetTypes.CUSTOM_MODEL, AssetTypes.MLFLOW_MODEL]
+PROPERTIES = ["commit_hash", "model_size"]
```

Comment:
language, license, dataset, sha, last_modified

cc; @ManojBableshwar, @skasturi

Reply:
Not needed in registration component. Resolving here